### DISCUSSION TITLE
RabbitMQ Activity - is there anyway to configure manual/auto ack ?

### DISCUSSION THREAD

**Discussion by jkeeley2073** [elsa-workflows/elsa-core#4439] [Q&A]
Just getting started with a POC and have a very basic workflow that is triggered by a rabbmq message and it works as expected except I don't see a way for me to configure how I'd like to handle acks

**Comment by sfmskywalker** [comment]
Hi @jkeeley2073 

I'm not too familiar with the RabbitMQ package myself, but from what I see, there's no way to configure how to handle acks.
If you want, I suggest that you clone the elsa-core project and explore the `Elsa.Activities.RabbitMq` project a bit, and perhaps propose a PR exposing the handling of acks, perhaps as a setting from the activity itself, or as an application-level setting, or otherwise.
